testing/dv3d/TestManager.py

```diff
@@ -80,7 +80,7 @@ def __init__( self, name, **args ):
         self.image_name = os.path.join( self.test_dir, 'images', '.'.join( [ self.name, 'png' ] )  )
         filename = args.get( 'file', DefaultSampleFile )
         self.varnames = args.get( 'vars', [ DefaultSampleVar ] )
-        self.file_path = os.path.join( vcs.sample_data, filename )
+        self.file_path = os.path.join(*["/"]+vcs.__path__[0].split("/")[:-4]+["share","uvcdat","sample_data",filename] )
```

Comment:
@aashish24 just to clarify setup.py puts the file in Library/Frameworks/Python/Version/2.7/share/uvcdat on mac. 
On Linux it is in share/uvcdat... There are two possibilities

* make our share/sample_data consistent with setup.py (different location between mac and linux)
* move @ThomasMaxwell sample data with our other sample data

I'm going to go with the later one.